tests/messages/properties/complex/test_request.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from tol_lab_share import error_codes
+from tol_lab_share.messages.properties.complex.request import Request
+from tol_lab_share.messages.properties.simple import Value
+
+
+@pytest.fixture
+def valid_request(request):
+    return Value(
+        {
+            "costCode": "CostCode789",
+            "libraryType": "LibraryType123",
+            "studyUuid": b"12345678-1234-1234-1234-1234567890ab",
```

Comment:
Should `studyUuid` be a bytes or string? `build_bioscan_pool_xp_msg` returns a nested request hash, it is converted to string there.